client/my-sites/customer-home/main.jsx

```diff
@@ -185,31 +185,58 @@ class Home extends Component {
 			checklistMode,
 			siteId,
 			currentThemeId,
+			siteIsUnlaunched,
+			isAtomic,
 		} = this.props;
 
+		// Show a thank-you message 30 mins post site creation/purchase
 		if ( isNewlyCreatedSite && displayChecklist ) {
+			if ( siteIsUnlaunched || isAtomic ) {
+				//Only show pre-launch or for Atomic sites
+				return (
+					<React.Fragment>
+						{ siteId && 'theme' === checklistMode && <QueryActiveTheme siteId={ siteId } /> }
+						{ currentThemeId && (
+							<QueryCanonicalTheme themeId={ currentThemeId } siteId={ siteId } />
+						) }
+						<img
+							src="/calypso/images/signup/confetti.svg"
+							aria-hidden="true"
+							className="customer-home__confetti"
+							alt=""
+						/>
+						<FormattedHeader
+							headerText={
+								this.props.siteHasPaidPlan
+									? translate( 'Thank you for your purchase!' )
+									: translate( 'Your site has been created!' )
+							}
+							subHeaderText={ this.getChecklistSubHeaderText() }
+						/>
+					</React.Fragment>
+				);
+			}
+		}
+
+		// Show a congratulatory message 30 mins post-launch
+		if ( ! siteIsUnlaunched && true ) {
 			return (
 				<React.Fragment>
-					{ siteId && 'theme' === checklistMode && <QueryActiveTheme siteId={ siteId } /> }
-					{ currentThemeId && <QueryCanonicalTheme themeId={ currentThemeId } siteId={ siteId } /> }
 					<img
 						src="/calypso/images/signup/confetti.svg"
 						aria-hidden="true"
 						className="customer-home__confetti"
 						alt=""
 					/>
 					<FormattedHeader
-						headerText={
-							this.props.siteHasPaidPlan
-								? translate( 'Thank you for your purchase!' )
-								: translate( 'Your site has been created!' )
-						}
-						subHeaderText={ this.getChecklistSubHeaderText() }
+						headerText={ translate( 'You launched your site!' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'You launched your site' )` *ES Score: 11*